docs/controls/file-pickers/js-v72/open-file.md

```diff
@@ -65,6 +65,7 @@ var odOptions = {
 | **clientId**        | The application ID generated by the app registration console for your integration.                                                                                                                                                                                                            |
 | **action**          | The action type being performed with the files selected. You can specify `share` to generate a sharable URL, `download` to receive a short-lived URL that downloads the content of the files, or `query` to return identifiers that can be used with the Microsoft Graph API or OneDrive API. |
 | **multiSelect**     | The default value is **false**, which requires the user to select a single file only, or **true** to enable the user to select multiple files.                                                                                                                                                |
+| **viewType** | The type of item which can be selected. The default value is `files`. You can specify `folders` to limit selection to _only_ folders or specify `all` which enables the selection of _both_ files and folders. |
```

Comment:
Looks good! I'm not sure if it's just me, but I would probably write "The type of item **that** can be selected."

Reply:
I think you're correct here 😃 👍